### ISSUE TITLE
Links to the examples section are broken

### ISSUE THREAD

**Issue by francbartoli** [opengeospatial/OGC-API-Maps#41]
Links in [https://github.com/opengeospatial/OGC-API-Maps/tree/master/core#examples](https://github.com/opengeospatial/OGC-API-Maps/tree/master/core#examples) are mostly broken or lead to a private resource

**Comment by joanma747** [comment]
Sorry for that. There are some mistakes in the links but the more dramatic thing is that I have lost control of my domain files at the end of my billing period on my swaggerhub account.. I have renewed it but the new plans only allow for 1 domain files that the company has block them all. I do not have updated copies of them so for the moment I have lost them. I'm paying 450$ a year and I have lost control of my files!!. This is how the cloud works.. I have asked for support and I hope this is fixed in time for the Sprint. This is a bit embarrassing.

**Comment by ghobona** [comment]
@joanma747 In the meantime, I propose to do this:

1. I will move the swagger hub examples out of the Examples section of the README.md and into a separate file in the same repo.
2. I will then publish one of the examples using ReDoc and GitHub Pages and link to it from the README.md file.

**Comment by ghobona** [comment]
Just realised that the API examples are working on Swagger Hub, even though the Domain is not accessible. So there is no need to change to ReDoc.

**Comment by joanma747** [comment]
Not exactly. All the dependencies are broken. It this point I'm googling pages on anger management!!. It is so frustrating!.

**Comment by joanma747** [comment]
I was able to circumvent the problem and now the two main examples work:

https://app.swaggerhub.com/apis/UAB-CREAF/ogc-api-maps-opf-xmp-more-1-collection/1.0.0
https://app.swaggerhub.com/apis/UAB-CREAF/ogc-api-map-tiles-opf-xmp-mt-more-1-collection/1.0.0

The trick was duplicate an old copy that I had of all domain files as api files. The swaggerhub environment does not know about that and it works!

**Comment by jerstlouis** [comment]
@joanma747 The README still includes the UAB-CREAF examples on SwaggerHub, while the [OpenAPI directory](https://github.com/opengeospatial/ogcapi-maps/tree/master/openapi) points to the YAML files and bundled JSON directly from this GitHub repository.

Perhaps we can remove the Examples section of the README now, since this is well explained in OpenAPI directory?

It would also be valuable to add to the example servers additional implementations besides our own in the example OpenAPI definition, e.g. if CREAF has a Maps server available that could be used for demonstrations within the Swagger UI client

**Comment by jerstlouis** [comment]
Addressed updating the examples in https://github.com/opengeospatial/ogcapi-maps/commit/688df28bb7c91b3eb5b98aee67b1abebf0b71f94 .
